CHANGELOG.md

```diff
@@ -25,14 +25,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Cursors are now inverted when their fixed color is similar to the cell's background
 - Use working directory of active process instead of shell for SpawnNewInstance action
 - Fallback to normal underline for unsupported underline types in `CSI 4 : ? m` escapes
+- Yellow/Red colors for message bar not being picked from user's config
```

Comment:
```suggestion
- Use yellow/red from the config for error and warning messages instead of fixed colors
```

We should rephrase this, since it's a change not a fix.